packages/web/src/pages/SignIn/SignIn.tsx

```diff
@@ -1,28 +1,39 @@
 import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { Container, Header, Footer, SignInForm } from 'components';
+
+import { Container, Header, Footer, SignInForm, FbAuth } from 'components';
 import { Link } from 'react-router-dom';
 import { AUTH_LOGIN } from 'store/auth/actions';
 import { getAuth } from 'store/auth/selectors';
 
+import './styles.scss';
+
 export const SignIn = () => {
   const dispatch = useDispatch();
   const { isLoading, error } = useSelector(getAuth);
 
   const handleSubmit = (email: string, password: string) => {
     dispatch({
       type: AUTH_LOGIN,
-      payload: { email, password },
+      payload: { email, password }
     });
   };
 
   return (
     <Container>
       <Header />
-      <SignInForm isLoading={isLoading} error={error} onSubmit={handleSubmit} />
+      <div className="container signInConrollersContainer">
```

Comment:
You're still using a `container` class which comes from Bootstrap, and has hidden behaviours, e.g. mediaQueries. Now we need to remember about this, in case it start to behave weirdly.

Reply:
It looks like you should use row instead.